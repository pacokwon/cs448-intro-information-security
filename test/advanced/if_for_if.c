int main()
{
    int x = 3;
    int i = 0;
    int j = 0;

    if(x >= 0)
    {
        for(i = 0; i < x; i++)
        {

            for(j = 0; j < i; j++)
            {
                int m = (i * j) / i;

                if(m >= 0)
                {
                    int k = x / m;
                }
            }
        }
    }
    else
    {
        x = -x;

        for(i = 0; i < x; i++)
        {

            for(j = 0; j > i; j--)
            {
                int m = (i * j) / i;

                if(m >= 0)
                {
                    int k = x / m;
                }
            }
        }
    }

}
