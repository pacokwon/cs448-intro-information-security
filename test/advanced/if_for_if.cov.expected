7,10
7,8
9,13
9,22
9,9
12,17
12,26
12,13
21,9
9,28
9,9
9,22
9,9
12,17
12,26
12,13
14,28
14,33
16,22
16,20
